<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>342</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QLabel" name="pol_show">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Полином:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="add_el">
      <property name="text">
       <string>Добавить элемент</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="add_el_re"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLineEdit" name="add_el_lm"/>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="chng_coef">
      <property name="text">
       <string>Изменить коэффициент</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="chng_coef_re"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="chng_coef_lm"/>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="del_el">
      <property name="text">
       <string>Удалить элемент по индексу</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="del_el_n"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="find_val">
      <property name="text">
       <string>Рассчитать значение в точке:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="find_val_re"/>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="find_val_lm"/>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QLabel" name="x_show">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Значение в точке x: </string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="change_data">
      <property name="text">
       <string>Изменить тип данных</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="5">
     <widget class="QLabel" name="type_show">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текущий тип данных:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
